package.json

```diff
@@ -1,27 +1,26 @@
 {
   "name": "mojito-js-delivery",
-  "version": "2.4.1",
+  "version": "2.4.2",
```

Comment:
Is it possible to bump this to 2.5.0? This is quite a chunky enhancement and has some big changes to the CLI and Pipelines.

Reply:
@kingo55 
Done.